plugin.video.channelsdvr/resources/settings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
It would be nice if the settings were in the new format.  Ronie made a script that does the conversion for you, and it's worked for almost every one of my addons.

https://forum.kodi.tv/showthread.php?tid=350135

Reply:
I agree, however, I haven't had the time to learn the new format, During my last attempt I encountered too many bugs; When it becomes mainstream I'll make the move.